src/UltraSinger.py

```
@@ -655,8 +657,30 @@ def infos_from_audio_video_input_file() -> tuple[str, str, str, MediaInfo]:
+    try:
+        audio_file = eyed3.load(settings.input_file_path)
+        if audio_file is not None and audio_file.tag is not None:
+            file_artist = audio_file.tag.artist or artist or "Unknown Artist"
+            file_title = audio_file.tag.title or title
+            if audio_file.tag.best_release_date:
+                year = audio_file.tag.best_release_date.year
+            if audio_file.tag.genre:
+                genres = [audio_file.tag.genre.name]
+            if audio_file.tag.images:
+                cover_image_data = audio_file.tag.images[0].image_data
+    except Exception:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)